cribl_stream/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2ef15aea-af91-4769-940c-2b124e4d04a6",
+  "app_id": "cribl-stream",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect observability data in a vendor-neutral data telemetry pipeline",
+    "title": "Cribl Stream",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cribl Stream Dashboard for Datadog",
+        "image_url": "images/cribl_data_dog_dashboard.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "Observability Changes Everything",
+        "image_url": "images/observability_changes.png",
+        "vimeo_id": 567294419
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Submitted Data Type::Metrics",
+      "Category::Metrics",
```

Comment:
Are there any other categories that are relevant to your integration? You can find the full list [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).

Reply:
Added more categories :
      "Category::AWS",
      "Category::Containers",
      "Category::Google Cloud",
      "Category::Cloud",
      "Category::Azure",
      "Category::Kubernetes",
      "Category::Log Collection",
      "Category::Security"